Import/SSRSDateSelector/Import/Method/Report_date_select.xml

```diff
@@ -29,10 +29,10 @@ var callback = function(res)
         var url   = aras.getServerBaseURL() + "RSGateway.aspx?irs:Report=" + reportName;      
         
         //Date Variables in this example is fromDate and toDate
-        url += "&amp;np:fromDate=" + fromDate; 
-        url += "&amp;np:toDate=" + toDate;
-        url += "&rs:Format=EXCEL";
-
+        url += "&np:fromDate=" + fromDate; 
+        url += "&np:toDate=" + toDate;
+        //url += "&rs:Format=EXCEL"; //output to Excel
```

Comment:
Why is this commented out?

Reply:
It's a toggle between outputting to Excel (this line) and outputting to PDF (next line); both can't be active at the same time, and I didn't event know PDF was an option until I went looking so I provided both. The user can then decide the output method based on their report structure. I set PDF as the default as that seemed the safest general output method.

Perfect world we could do this via dropdown box or something but that was outside the scope of this update.